lib/fetch/webidl.js

```diff
@@ -273,8 +273,8 @@ webidl.recordConverter = function (keyConverter, valueConverter) {
     const result = {}
 
     if (!types.isProxy(O)) {
-      // Object.keys only returns enumerable properties
-      const keys = Object.keys(O)
+      // 1. Let desc be ? O.[[GetOwnProperty]](key).
+      const keys = [...Object.getOwnPropertyNames(O), ...Object.getOwnPropertySymbols(O)]
```

Comment:
Although Object.keys returns enumerable *string* properties, it didn't include enumerable *symbol* properties. This is only noticeable when doing something along the lines of `new Headers({ [Symbol.iterator]: null })`.